packages/ui/src/components/Transactions/TransactionProgress.tsx

```diff
@@ -0,0 +1,128 @@
+import LinearProgress, { LinearProgressProps } from '@mui/material/LinearProgress'
+import { styled } from '@mui/material/styles'
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+  Box,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon
+} from '@mui/material'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import HourglassBottomIcon from '@mui/icons-material/HourglassBottom'
+import AccountDisplay from '../AccountDisplay'
+import CheckIcon from '@mui/icons-material/Check'
+
+interface TransactionProgressProps {
+  value: number
+  threshold: number
+  possibleSigners: string[]
+  approvals: string[]
+}
+const ListItemToSign = ({ approvals, signer }: { approvals: string[]; signer: string }) => {
+  return (
+    <ListItem disablePadding>
+      <ListItemButton>
+        <ListItemIcon>
+          {approvals.includes(signer) ? <CheckIconStyled /> : <HourglassBottomIcon />}
```

Comment:
At first I thought, let's show only the ones who didn't sign, but I guess it's the same info more or less. Let's go with this. I must remember to ask users what they prefer and if that works.